Body/AAUHuman/Trunk/JointsCervical.any

```diff
@@ -219,8 +219,7 @@ AnyFolder Rotation = {
    
    RhythmDriverLinear rhythmC2C1SkullThoraxRotation( _REDEFINE_VARIABLES=On) = 
    {
-     // The -1 is because C2C1 is a revolute jnt and has the opposite signs for output.
-     AnyVector RhythmCoefficients ??= {132/77, -1};
+     AnyVector RhythmCoefficients ??= {132/77, 0.7967};
```

Comment:
```suggestion
     AnyVector RhythmCoefficients ??= {0.2721+0.3061, 0.2721};
```

Reply:
Does this suggestion give the same as your tweak? With these numbers it will be easier to see where the numbers come from in the excel sheets where the rhythm coefficients are listed. 

![image](https://github.com/user-attachments/assets/412b8b09-48f6-4d36-b5b2-76dba7f9bb35)

Reply:
It's super close but a bit worse (but this could be the particular case I tested). I would say let's keep your suggestion because the numbers are traceable, and it's still accurate (enough).
![image](https://github.com/user-attachments/assets/88a5666b-c870-4771-895c-8941261726bd)